src/pages/zh-CN/install/auto.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 通过自动命令行来安装Astro
+description: 如何通过 create-astro CLI 工具使用 NPM、PNPM 或 Yarn 安装 Astro。
+layout: ~/layouts/MainLayout.astro
+setup: import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+---
```

Comment:
Oh, we don’t need to include `i18nReady` tags in translations — that’s there in the English version so that the page displays in https://github.com/withastro/docs/issues/438, but sharp eyes!